tests/test-verify-pool-md.yml

```diff
@@ -3,9 +3,8 @@
   # Only when pool is raid
 - name: check RAID options
   block:
-
     - name: get information about RAID
-      command: "mdadm --detail /dev/md/{{ storage_test_pool.name }}-1"
+      command: "mdadm --detail /dev/md/{{ storage_test_pool.raid_name }}"
```

Comment:
We already have code to list and iterate over the PVs in `tests/test-verify-pool-members.yml`.  Also see `tests/test-verify-pool-members-encryption.yml`. Wouldn't it be simpler to use this model to validate the raid settings?

Reply:
I have moved the code so it is called from `test-verify-pool-members.yml`